### PR TITLE
Rename `GetUnits` to `LoadUnits`

### PR DESCRIPTION
**Pull request by yp05327**
Same as https://github.com/go-gitea/gitea/pull/22967

### CODE REVIEW

**Review by delvh — APPROVED** [review]

File: routers/api/v1/org/team.go [hunk 1]
```diff
@@ -256,7 +256,7 @@ func EditTeam(ctx *context.APIContext) {
 
 	form := web.GetForm(ctx).(*api.EditTeamOption)
 	team := ctx.Org.Team
-	if err := team.GetUnits(); err != nil {
+	if err := team.LoadUnits(ctx.Context); err != nil {
```
Comment (lunny) [inline routers/api/v1/org/team.go]: I think just `ctx` is OK.

Comment (yp05327) [reply routers/api/v1/org/team.go]: Fixed in https://github.com/go-gitea/gitea/pull/22970/commits/c40582704303c30c3481c57bf3c561eb0235afce

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LG-TM bot

File: routers/api/v1/org/team.go (hunk 1 above)
Comment (delvh) [reply routers/api/v1/org/team.go]: Ah, interesting.
I saw that as well, but from my look in the code, I didn't know whether it implements `context.Context`, so I accepted it.

**Pull request merged**